src/types.ts

```diff
@@ -8,6 +8,7 @@ export type Config = {
   interval: number;
   onTimeOver?: () => void;
   onTimeUpdate?: (time: number) => void;
+  shouldAutostart: boolean;
```

Comment:
isn't `autostart` better than `shouldAutostart`?

Reply:
I prefer use `has`, `should`, `is`, ... prefixes for boolean variables. Why do you think it's better to name it `autostart`?

Reply:
I think `autostart: true` is describable enough, but it is up to you.

Reply:
I'll make some concessions ;-) It's sometimes a hard task to define some variable names... Let's use `autostart`, it's shorter.

Reply:
i'll make the change soon.